tests/test_functional.py

```diff
@@ -216,6 +216,26 @@ def test_valid_import_with_multiple_rc(self, input_file, user, testapp):
         DonorsOverview.query.get("205225299").donation_count_total == 70
         DonorsOverview.query.get("1860231599").donation_count_total == 6
 
+    def test_zero_donations(self, user, testapp):
+        # three lines in the input file end with zero and should
+        # be automatically ommited from the import
+        ends_with_zero = 3
+        input_data = Path("tests/data/valid_import_zeroes.txt").read_text()
+        new_records = len(input_data.strip().splitlines()) - ends_with_zero
+        existing_records = Record.query.count()
+        existing_batches = Batch.query.count()
+
+        login(user, testapp)
+        res = testapp.get(url_for("donor.import_data"))
+        form = res.form
```

Comment:
Je toto nutné?

Reply:
Není, ale zkracuje to následující dva řádky.